server/schedulers/evict_leader.go

```diff
@@ -348,7 +348,7 @@ func scheduleEvictLeaderOnce(name, typ string, cluster schedule.Cluster, conf ev
 			log.Debug("fail to create evict leader operator", errs.ZapError(err))
 			continue
 		}
-		op.SetPriorityLevel(core.HighPriority)
+		op.SetPriorityLevel(core.High)
```

Comment:
I think the priority of `evict leader` should be urgent, It should be higher than the hot region.

Reply:
make sense